Tavisca.Bootcamp.LanguageBasics.Exercise2/Program.cs

```diff
@@ -6,25 +6,175 @@ public static class Program
     {
         static void Main(string[] args)
         {
-            Test(new[] {"12:12:12"}, new [] { "few seconds ago" }, "12:12:12");
+            Test(new[] { "12:12:12" }, new[] { "few seconds ago" }, "12:12:12");
             Test(new[] { "23:23:23", "23:23:23" }, new[] { "59 minutes ago", "59 minutes ago" }, "00:22:23");
             Test(new[] { "00:10:10", "00:10:10" }, new[] { "59 minutes ago", "1 hours ago" }, "impossible");
             Test(new[] { "11:59:13", "11:13:23", "12:25:15" }, new[] { "few seconds ago", "46 minutes ago", "23 hours ago" }, "11:59:23");
             Console.ReadKey(true);
         }
 
-        private static void Test(string[] postTimes, string[] showTimes, string expected)
-        {
+        private static void Test(string[] postTimes, string[] showTimes, string expected)
+        {
             var result = GetCurrentTime(postTimes, showTimes).Equals(expected) ? "PASS" : "FAIL";
             var postTimesCsv = string.Join(", ", postTimes);
             var showTimesCsv = string.Join(", ", showTimes);
-            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
+            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
         }
 
-        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
-        {
+        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+            //throw new NotImplementedException();
+            string min = "00:00:00", max = "23:59:59";
+            for (int i = 0; i < exactPostTime.Length; i++)
+            {
+                string back = "";
+                string present = "";
+                int hours, minutes, seconds;
+                hours = Convert.ToInt32(exactPostTime[i].Substring(0, 2));

```

Comment:
use split to extract hours, min, seconds

Reply:
sure..